### PR TITLE
Working group detail

### PR DESCRIPTION
**Pull request by jhonny-m**

### CODE REVIEW

File: packages/frontend-utils/package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
   },
   "peerDependencies": {
     "react": "^17.0.2",
-    "react-router-dom": "^5.2.0"
+    "react-router-dom": "^5.2.0",
```
Comment (ricardocarvalh0) [inline packages/frontend-utils/package.json:17]: is this a dependency we don't want to pin?

File: packages/gp2-components/src/molecules/CardWithBackground.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+type CardWithBackgroundProps = {
+  image: string;
+};
+
+const containerStyles = (image: string) =>
+  css({
+    backgroundImage: `url(${image})`,
+    borderRadius: `8px`,
+    backgroundSize: '100%',
+    padding: '16px',
```
Comment (ricardocarvalh0) [inline packages/gp2-components/src/molecules/CardWithBackground.tsx:13]: shouldn't padding be `perRem` ?

**Review by ricardocarvalh0 — APPROVED** [review]

File: packages/frontend-utils/package.json (hunk 1 above)
Comment (jhonny-m) [reply packages/frontend-utils/package.json:17]: to pin? what do you mean?

Comment (ricardocarvalh0) [reply packages/frontend-utils/package.json:17]: We have this change recently where it removes `^` from versions. https://github.com/yldio/asap-hub/commit/c4e964bf771f6423f4c692088b864c9764dc8927
I wonder if it should be the same for react-router-dom

**Review by oratiu — APPROVED** [review]

**Pull request merged**